streamlit_ace/__init__.py

```diff
@@ -113,13 +117,16 @@ def st_ace(
         annotations=annotations,
         markers=markers,
         name=key or "ace-editor",
-        key=key or "ace-editor"
+        key=key,
+        default=str(value),
```

Comment:
Pass the default value to the special "default" parameter, so that the component returns the proper value when it's first instantiated. Otherwise, the component will return None upon instantiation, even though it has text in it.